### DISCUSSION TITLE
how to set auth.jwt.secret in kubernetes if plugin emqx_auth_jwt is enabled

### DISCUSSION THREAD

**Discussion by microservicesengineer** [emqx/emqx#7272] [General]
### Subject of the support
I have setup a emqx cluster in kuberntes, and I want to enable emqx_auth_jwt plugin. I want to change the secret as below, where can i change the configuration? Thanks.

##
## The key to verify the JWT Token using HMAC algorithm
auth.jwt.secret = emqxsecret

Error/Warning printout if any. 

### Your environment

For EMQ X 4.3 or newer, please provide the log archive generated by [node_dump tool](https://github.com/emqx/emqx/blob/master/bin/node_dump) 

Otherwise please provide below info:
- EMQ 4.4.1 version 
- Others:

### LOG File

Missing log file can delay the handling of the issue.

**Comment by zmstone** [comment]
@Rory-Z could you help to have a look

**Comment by Rory-Z** [comment]
Hi, @microservicesengineer  You can modify the configuration by setting environment variables, see: https://www.emqx.io/docs/en/v4.4/configuration/environment-variable.html
